LICENCE-header.txt

```diff
@@ -0,0 +1,17 @@
+====
+    This file is part of obelix.
```

Comment:
The maven plugin will ensure that the file is included inn all scripts, so no worries.